x/protorev/keeper/epoch_hook.go

```diff
@@ -39,6 +39,12 @@ func (h EpochHooks) AfterEpochEnd(ctx sdk.Context, epochIdentifier string, epoch
 	if h.k.GetProtoRevEnabled(ctx) {
 		switch epochIdentifier {
 		case "day":
+			// Calculate and distribute protorev profits
+			err := h.CalculateDistributeProfits(ctx)
```

Comment:
Renaming here too to match the other rename
```suggestion
			err := h.CalculateAndDistributeProfits(ctx)
```

Reply:
Good idea updated here => https://github.com/osmosis-labs/osmosis/pull/8106/commits/e043b86ebc451eb183f8de6fc70e26955c1fb29d